src/ipc-handlers.ts

```diff
@@ -1267,6 +1268,15 @@ export async function isCATrusted(): Promise< boolean > {
 	return isRootCATrusted();
 }
 
+export async function trustCertificate(): Promise< void > {
+	const platform = process.platform;
+	if ( platform === 'win32' ) {
+		return trustRootCA();
```

Comment:
I think we should consider returning a boolean from `trustRootCA` to indicate if the operation was successful. If it fails, we can display an error modal. Silently failing would not be very helpful for users.

Reply:
That's a very good suggestion. I have addressed it.